plugins/modules/win_ntp.ps1

```diff
@@ -0,0 +1,183 @@
+#!powershell
+
+# Copyright: (c) 2020, Sakar Mehra <sakarmehra100@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+
+$ErrorActionPreference = "Stop"
+
+$params = Parse-Args $args -supports_check_mode $true
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
+
+$peerlist = Get-AnsibleParam -obj $params -name "peerlist" -type "list"
+$enabled = Get-AnsibleParam -obj $params -name "enabled" -type "bool" -default $true
+$time_zone = Get-AnsibleParam -obj $params -name "time_zone" -type "str" -default "UTC"
+$factory_default = Get-AnsibleParam -obj $params -name "factory_default" -type "bool" -default $false
+$type = Get-AnsibleParam -obj $params -name "type" -type "str" -ValidateSet 'NTP','NT5DS','NoSync','AllSync'
+$cross_site_sync_flags = Get-AnsibleParam -obj $params -name "cross_site_sync_flags" -type "int" -ValidateSet 0 ,1, 2
+$sync_clock = Get-AnsibleParam -obj $params -name "sync_clock" -type "bool" -default $false
+
+$result = @{
+    changed = $false
+}
+
+$outputItem = [PsCustomObject]@{
+    ComputerNameFQDN = $null
+    SourceName = $null
+    SourceNameRaw = $null
+    LastTimeSyncElapsedSeconds = $null
+    LastTimeSynchronizationDateTime = $null
+    StatusRequiredSourceType= $null
+    StatusDateTime= $null
+    StatusLastTimeSynchronization = $null
+}
+
+$current_time_zone= [System.TimeZoneInfo]::Local.Id
+if ($null -ne $time_zone -and $current_time_zone -ne $time_zone) {
```

Comment:
We already have a timezone module, we shouldn't overload that functionality here.

Reply:
I was thinking, timezone functionality should also be present in this module too, I am aware of the `win_timezone` module but it's easier for users to set time while configuring the NTP.